types/experiments.ts

```diff
@@ -0,0 +1,81 @@
+/** All fields are publically accessible via the API and available to clients */
+interface Experiment {
+  /** Unique identifier for the experiment */
+  slug: string;
+  /** Publically-accesible name of the experiment */
+  name: string;
```

Comment:
In the past we had a field like this "description" with a comment that it would be public. No one ever read the comment, and put lots of not-public stuff in it. I recommend this mirrors `public_description` and is named `public_name`.